doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -188,6 +163,72 @@ preamble similar to the one below, indicating that this is an executable file:
 
 You don't need to edit this preamble, as it is autogenerated.
 
+Links, formatting and images
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When converting a notebook to a MyST Markdown file, you may need to adjust some
+of the content to match the MyST Markdown syntax.
+
+- **External links:** MyST Markdown uses the standard Markdown syntax for links.
+  For example, ``[link text](https://www.example.com)``. If you have a link in a
+  notebook cell, you can convert it to the MyST Markdown format by enclosing the
```

Comment:
What format would it start in such that a conversion would be necessary?

Reply:
It could be just a plain URL that gets highlighted as a link in a notebook.

Reply:
So in MyST markdown, https://www.example.com will render differently than it does here?

Reply:
I am not sure these links are automatically converted to a `<a href` element, I should check that.

Reply:
Ok - I just added linkify-it-py to the requirements list and we don't need to worry about this anymore.